src/utils/string/index.ts

```diff
@@ -121,15 +122,19 @@ export const getDateCopy = (
     return "";
   }
   const { dateOnly, omitSeconds, omitTimezone, tz } = options || {};
-  let { dateFormat } = options || {};
+  let { dateFormat, timeFormat } = options || {};
   if (!dateFormat) {
     dateFormat = "MMM d, yyyy";
   }
+  if (!timeFormat) {
+    timeFormat = "h:mm:ss aa";
```

Comment:
nit. could we reuse the TimeFormat enum declared above instead of duplicating here?